platform/kube/inject/inject.go

```diff
@@ -190,6 +193,20 @@ func GetInitializerConfig(kube kubernetes.Interface, namespace, injectConfigName
 		return nil, err
 	}
 
+	if c.IncludeNamespaces != nil && c.ExcludeNamespaces != nil {
+		return nil, fmt.Errorf("cannot configure both namespaces and excludeNamespaces")
```

Comment:
error log needs to be updated to includeNamespaces

Reply:
I want the log show key word in the configuration file, and we are still using `namespace` in configuration file to keep backward compatibility.